parsel/csstranslator.py

```diff
@@ -91,12 +98,18 @@ def xpath_text_simple_pseudo_element(self, xpath):
 
 
 class GenericTranslator(TranslatorMixin, OriginalGenericTranslator):
-    pass
-
+    @lru_cache(maxsize=256)
+    def css_to_xpath(self, css, prefix='descendant-or-self::'):
```

Comment:
I think it would be nice to add caching without repeating code - is it possible to call parent method, or are there any differences?

Reply:
You're right @lopuhin , I just changed the code so it calls the parent method instead. Please let me know if that can resolve this problem!